blueprints/data-solutions/data-platform-foundations/03-orchestration.tf

```diff
@@ -44,6 +49,8 @@ module "orch-project" {
       "roles/iam.serviceAccountUser",
       "roles/storage.objectAdmin",
       "roles/storage.admin",
+      "roles/artifactregistry.admin",
```

Comment:
I would create also a dedicated SA with roles needed to build the template. This is useful to create a CI/CD integration.

I would add a note in the README on how this can be integrated with CI/CD rather than using this with manual commands.

Reply:
Dedicated SA for cloud build created and tested